Jenkinsfile

```diff
@@ -28,6 +28,22 @@ pipeline {
         }
         stage('Build deb/rpm') {
             stages {
+                // Replace the pkgrel value with the git commit hash to ensure that
+                // each merged PR has unique artifacts and to prevent conflicts between them.
+                // Note that the pkgrel value will remain as it was in the codebase to avoid
+                // conflicts between multiple open PRs
+                stage('Add timestamp and commit hash') {
+                    when {
+                        branch 'develop'
+                    }
+                    steps {
+                        sh'''
+                            export TIMESTAMP=$(date +%s)
+                            export GIT_COMMIT_SHORT=$(git rev-parse HEAD | head -c 8)
+                            sed -i "s/pkgrel=\\".*\\"/pkgrel=\\"$TIMESTAMP+$GIT_COMMIT_SHORT\\"/" ./package/PKGBUILD
```

Comment:
you can avoid sed and use a `yap` flag.

```
  -r, --pkgrel string         Use a custom package release (pkgrel)
  -w, --pkgver string         Use a custom package version (pkgver)
```

Reply:
Thanks for the suggestion. We already saw the closed PRs with your solution, but we prefer to have a separated stage that runs only when the branch is "develop" to avoid a conditional check.